resources/testharness.js

```diff
@@ -1669,6 +1751,7 @@ policies and contribution forms [3].
         Object.keys(this).forEach(
                 (function(key) {
                     var value = this[key];
+                    if (key === '_done_callbacks' ) { return; }
```

Comment:
This looks a bit fishy, why was this needed here, when so many other properties are not?

Reply:
As an array containing function values, the value of the `_done_callbacks`
property is the only one that cannot be serialized for transfer between realms.
I agree that this has a little bit of code smell, though. Here are a few ideas
for how we could address it:

- Add an in-line comment explaining the condition
- Define the property as non-enumerable so that it isn't visited by
  `Object.keys`
- Define a property like `RemoteTest.UNCLONABLE = ["_done_callbacks"]` and
  reference that when creating a cloned instance

Do you have any preference?

Reply:
Is this the only bit of state on RemoteTest that doesn't need to be transferred? If so just a comment about how it isn't needed and why it can't be serialized would do. If there are other bits that aren't needed, excluding them all would make sense. Couldn't something more like `Test.prototype.structured_clone` work, where the needed bits are explicitly listed?

Reply:
Optimistically, I would hope that everything is actually necessary. It's
definitely possible that's not true, but I'm reluctant to make a change like
that within this patch. I've added this comment instead:

>     // `RemoteTest` instances are responsible for managing
>     // their own "done" callback functions, so those functions
>     // are not relevant in other execution contexts. Because of
>     // this (and because Function values cannot be serialized
>     // for cross-realm transmittance), the property should not
>     // be considered when cloning instances.